app/src/main/java/com/lagradost/cloudstream3/syncproviders/providers/Subdl.kt

```diff
@@ -181,7 +181,7 @@ class SubDlApi(index: Int) : InAppAuthAPIManager(index), AbstractSubApi {
     }
 
     data class SubtitleOAuthEntity(
```

Comment:
I did not see this before, but if you are storing this as a the SUBDL_SUBTITLES_USER_KEY then it *needs* JsonProperty